pkg/state/state.go

```diff
@@ -1976,6 +1985,12 @@ func (st *HelmState) DiffReleases(helm helmexec.Interface, additionalValues []st
 				flags := prep.flags
 				release := prep.release
 				buf := &bytes.Buffer{}
+
+				suppressDiff := suppressDiff
```

Comment:
what's the meaning? same variable name?

Reply:
From go perspective, its a new variable with an different scope. This prevents that once the variable hits true inside the loop, the value of the var is false again on the next item loop.

Reply:
I changed the variable name on the left side, I hope its less confusing for know.

I need a local variable here, because otherwise I would modify suppressDiff for all releases.

If `suppressDiff = true` hits once, all other releases in this for loop will have, `suppressDiff = true` which might be not correct.

Reply:
in Golang. This is just a copy of the value and doesn't affect the original. WDYT? @xiaomudk

Reply:
Yeah

Reply:
so do we need to assign a new var? @jkroepke

Reply:
Yes, as you mentioned, to not affect the original one. Otherwise, defining suppressDiff on one release would affect other releases, too.

Reply:
@yxxhero @jkroepke I think there is no problem now. Thinks.